src/AudioUnit/AudioComponent.cs

```diff
@@ -342,6 +351,13 @@ public UIKit.UIImage GetIcon (float desiredPointSize)
 				return AudioComponentGetLastActiveTime (handle);
 			}
 		}
+
+		[Watch (7,0), TV (14,0), iOS (14,0)]
```

Comment:
API diff for watchOS is empty so this code is not included in the platform assembly
Either it's available and should be added or the `[Watch (7,0)]` attribute should be removed (here and elsewhere).
Considering that there was no xtro file updated for the watchOS profile I suspect it's the former.